docs/Deploy/overview.mdx

```diff
@@ -20,12 +32,47 @@ Where can you deploy conduit?
 After you install Docker, you can pull the Conduit image by running the following command:
 
 ```shell
-docker pull ghcr.io/conduitio/conduit:latest
+$ docker pull ghcr.io/conduitio/conduit:latest
 ```
 Then, you can run Conduit by running the following command:
 
 ```shell
-docker run -d -p 8080:8080 ghcr.io/conduitio/conduit
+$ docker run -d -p 8080:8080 ghcr.io/conduitio/conduit
+```
+
+**Helpful Tip:** If you're using Docker, check out [Watchtower](https://containrrr.dev/watchtower/). After we push an update, Watchtower will keep your Conduit container updated by gracefully shutting down your existing container and restarting it with the same options used when initially deployed.
+
+## Building from source
+
+Before you can build Conduit from source, you need to have Go 1.18 or higher installed. The Go website has [comprehensive instructions](https://go.dev/doc/install) on how to get the language installed on your machine. Once installed, you can follow the rest of the instructions.
```

Comment:
How about the UI-related stuff? https://github.com/ConduitIO/conduit#build-from-source

Reply:
I can mention it.